src/plugins/home/public/application/components/homepage/sections/learn_basics.tsx

```diff
@@ -0,0 +1,165 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React, { FC } from 'react';
+import {
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiCard,
+  EuiIcon,
+  EuiListGroup,
+  EuiListGroupItemProps,
+  IconType,
+} from '@elastic/eui';
+import { i18n } from '@osd/i18n';
+import { Section } from '../../../../services/section_type/section_type';
+import { renderFn } from './utils';
+import { getServices } from '../../../opensearch_dashboards_services';
+
+const Card: FC<{
+  iconType: IconType;
+  title: string;
+  listItems: EuiListGroupItemProps[];
+}> = ({ iconType, title, listItems }) => (
+  <EuiFlexItem grow={1}>
+    <EuiCard
+      icon={<EuiIcon type={iconType} size="xl" />}
+      title={
+        <>
+          {title} <EuiIcon type="popout" />
+        </>
+      }
+      layout="horizontal"
+    >
+      <EuiListGroup className="home-basics-listGroup" listItems={listItems} />
+    </EuiCard>
+  </EuiFlexItem>
+);
+
+const render = renderFn(() => {
+  const services = getServices();
+  const logos = services.chrome.logos;
+
+  const gettingStartedLinks: EuiListGroupItemProps[] = [
+    {
+      label: i18n.translate('home.sections.learnBasics.gettingStarted.quickstart', {
+        defaultMessage: 'OpenSearch Dashboards quickstart',
+      }),
+      href: 'https://opensearch.org/docs/latest/dashboards/quickstart/',
+      target: '_blank',
+    },
+    {
+      label: i18n.translate('home.sections.learnBasics.gettingStarted.dataVis', {
+        defaultMessage: 'Building data visualizations',
+      }),
+      href: 'https://opensearch.org/docs/latest/dashboards/visualize/viz-index/',
+      target: '_blank',
+    },
+    {
+      label: i18n.translate('home.sections.learnBasics.gettingStarted.dashboards', {
+        defaultMessage: 'Creating dashboards',
+      }),
+      href: 'https://opensearch.org/docs/latest/dashboards/dashboard/index/',
+      target: '_blank',
+    },
+  ];
+
+  const dataDiscoveryLinks: EuiListGroupItemProps[] = [
+    {
+      label: i18n.translate('home.sections.learnBasics.dataDiscovery.getFamiliar', {
+        defaultMessage: 'Get familiar with Discover',
+      }),
+      href: 'https://opensearch.org/docs/latest/dashboards/discover/index-discover/',
+      target: '_blank',
+    },
+    {
+      label: i18n.translate('home.sections.learnBasics.dataDiscovery.runQueries', {
+        defaultMessage: 'Run queries in the Dev Tools Console',
+      }),
+      href: 'https://opensearch.org/docs/latest/dashboards/dev-tools/run-queries/',
+      target: '_blank',
+    },
+    {
+      label: i18n.translate('home.sections.learnBasics.dataDiscovery.indexes', {
+        defaultMessage: 'Working with indexes',
+      }),
+      href: 'https://opensearch.org/docs/latest/dashboards/im-dashboards/index/',
+      target: '_blank',
+    },
+  ];
+
+  const observabilityLinks: EuiListGroupItemProps[] = [
```

Comment:
imo: this should be configurable or make the sections are service that plugins can register themselves too. since not everyone who downloads OpenSearch Dashboards has observability plugin installed. I can imagine it being confusing to have links to documentation on the home page but if I do not have the plugin installed

Reply:
Yeah I agree but I haven't gotten updated mocks for a while, so I'm not sure what _should_ go here. I think for this PR, the main focus is the tech and getting things looking and working good from the API side. This would be a quick, easy, and non-breaking change to make in the future

Reply:
I think these are safe to be here because they link out to the docs and not to pages exposed by a plugin. I think I remember @BSFishy explaining that plugins can or will be able to register for stuff.